### PR TITLE
eight task Gontari Anna

### PR DESCRIPTION
**Pull request by Gontari-Anna**

### CODE REVIEW

File: src/main/java/anna_gontari/CopyingFile.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public static void copyFile() {
+        System.out.println("Please, enter file's path from which you want to copy information: ");
+        Scanner scanner = new Scanner(System.in);
+        String input1 = scanner.nextLine();
+        System.out.println("Please enter new file name:");
+        String input2 = scanner.nextLine();
+
+        File path = new File("New Folder/copy Files/new Files");
+        try {
+            if (path.mkdir() == true) {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/anna_gontari/CopyingFile.java:23]: path.mkdir() can be only true or false 

path.mkdir() returns true or false , so the third else is redundant, there is not 3-th case

File: src/main/java/anna_gontari/files/MyFile.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Welcome to the file
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/anna_gontari/files/MyFile.txt:1]: don't need to commit files as they will not be in use

**Review by dgrimaliuc-crunchyroll — CHANGES_REQUESTED** [review]

File: src/main/java/anna_gontari/CopyingFile.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
```
Comment (dgrimaliuc) [inline src/main/java/anna_gontari/CopyingFile.java:24]: make it easier 
if (path.mkdir())

File: src/main/java/anna_gontari/CopyingFile.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+                Files.copy(Path.of("New Folder/copy Files/new Files" + input1), Path.of("New Folder/copy Files/new Files" + input2));
+                System.out.println("Requested file was found and information was successfully copied");
+            } else if (path.mkdir() == false) {
```
Comment (dgrimaliuc) [inline src/main/java/anna_gontari/CopyingFile.java:27]: make it easier 
just else

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]

**Review by dgrimaliuc — APPROVED** [review]

**Pull request merged**